stf/src/sgx.rs

```diff
@@ -39,6 +39,7 @@ const ALICE_ENCODED: [u8; 32] = [
     212, 53, 147, 199, 21, 253, 211, 28, 97, 20, 26, 189, 4, 169, 159, 214, 130, 44, 133, 88, 133,
     76, 205, 227, 154, 86, 132, 231, 165, 109, 162, 125,
 ];
+const ALICE_FUNDS: Balance = 100000;
```

Comment:
This is way too little I'm afraid as fees are higher. I suggest a balance of 10^15

Reply:
Done